client/app/views/dashboard/repos/repos.html

```diff
@@ -40,6 +40,7 @@
                 <span ng-hide="repo.build" class="name">{{ repo.label }}</span>
                 <span ng-show="repo.build" class="name">
                   <a ui-sref="dashboard.ciBuild({ buildId: parseInt(repo.build.id) })">{{ repo.label }}</a>
+                  <a href="http://www.shoov.io/tutorials/lesson2-scaffold/" class="repo-error tutorial-link" ng-show="!repo.build.enabled && repo.build.disable_reason == 'no_config_file'" target="_blank">No .shoov.yml file.</a>
```

Comment:
@HelenaEksler this doesn't seem to work on repos that were not created yet - only on existing ones. (i.e. it will show only when we have `repo.build`)